cmd/main.go

```diff
@@ -201,6 +202,26 @@ func maxDuration(dur time.Duration, durations ...time.Duration) time.Duration {
 	return ret
 }
 
+func WrapServeError() func(rw http.ResponseWriter, r *http.Request, err error) {
+	compositeErrorDefaultMessage := "validation failure list:"
+	unsupportedHttpCodes := map[int32]struct{}{
+		http.StatusUnprocessableEntity: {},
+	}
+
+	return func(rw http.ResponseWriter, r *http.Request, err error) {
+		if e, ok := err.(*errormiddleware.CompositeError); ok {
```

Comment:
Is it ever not a CompositeError?

Reply:
All the errors are passing through this function since this is a wrapper, so yes.